src/index.js

```diff
@@ -174,6 +174,10 @@ const renderLevel = (root, levelNumber) => {
     if (soko.goto(row, col)) {
       renderBoard();
     }
+    const playerSequence = soko.sequence()[soko.sequence().length - 1];
```

Comment:
Maybe `lastMove` would be clearer--it's not really a `playerSequence`, just a single step.

If `soko.goto()` failed, then are we sure we're not indexing out of bounds here? (I'm just eyeballing here, correct me if I'm wrong)

Reply:
We can use the str.charAt(length - 1) to get the last character of a string. This method will return an empty string if it's out of bounds.